nrf_rpc/CHANGELOG.rst

```diff
@@ -9,6 +9,12 @@ Changelog
 
 All the notable changes to this project are documented on this page.
 
+Main branch
+***********
+
+* Enabled to initialize the nRF RPC more than once.
+  Subsequent initialization of an already properly initialized nrfRPC will succeed.
```

Comment:
```suggestion
  Subsequent initialization of an already properly initialized nRF RPC will succeed.
```

Reply:
Is there anything new in this sentence in comparison to the first one?